features/step_definitions/controllers-steps.js

```diff
@@ -92,6 +92,14 @@ Then('I should receive a result matching:', function (dataTable) {
   should(this.props.result).matchObject(expectedResult);
 });
 
+Then('I should receive a response matching:', function (dataTable) {
+  const expectedResult = this.parseObject(dataTable);
+
+  should(this.props.response).not.be.undefined();
```

Comment:
Nitpicking: this will generate a better error message instead of `expected undefined to not be undefined`
```suggestion
  should(this.props).have.property('response');
```